lib/stream.js

```diff
@@ -51,15 +51,32 @@ Stream.isReadable = utils.isReadable;
 Stream.Readable = require('internal/streams/readable');
 for (const key of ObjectKeys(streamReturningOperators)) {
   const op = streamReturningOperators[key];
-  Stream.Readable.prototype[key] = function(...args) {
-    return Stream.Readable.from(ReflectApply(op, this, args));
-  };
+  // name function
```

Comment:
lmk if you think this needs a longer comment or none at all (i.e. how "known" is this hack for setting a function's name?)

Reply:
Why would you need this hack at all in node? Function names are configurable.

Reply:
So it's ok/better to just `ObjectDefineProperty(fn, 'name', { value: op.name })` ? 

I don't remember where I saw that not working -_-

Reply:
Yes. In a package, where you need to support older engines, setting a function’s name is more complex, because it’s not configurable if you go back far enough. node doesn’t have that problem tho.

Reply:
ok thanks I'll refactor.